x-pack/plugins/data_visualizer/public/application/common/components/results_links/results_links.tsx

```diff
@@ -231,6 +264,19 @@ export const ResultsLinks: FC<Props> = ({
           onClick={showFilebeatFlyout}
         />
       </EuiFlexItem>
+      {additionalLinks
+        .filter(({ id }) => generatedLinks[id] !== undefined)
+        .map((link) => (
+          <EuiFlexItem>
+            <EuiCard
+              icon={<EuiIcon size="xxl" type={link.icon} />}
+              data-test-subj="fileDataVisLink-"
```

Comment:
nit: did you want to append an identifier to this test subject? or should we remove the `-` at the end?

Reply:
nope! https://github.com/elastic/kibana/pull/103932/commits/2097f0b5909719e3c8d65734672a615f55dad9f2